src/main/webapp/help/JFrogPlatformBuilder/help-allowHttpConnections.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Check to allow HTTP connections with the JFrog platform (Not recommended).
```

Comment:
Check to allow HTTP connections with the JFrog platform (Not recommended).
-->
Check to allow HTTP (non HTTPS) connections with the JFrog platform (Not recommended).